plugins/performance-lab/readme.txt

```diff
@@ -15,13 +15,14 @@ The Performance Lab plugin is a collection of features focused on enhancing perf
 
 The feature plugins which are currently featured by this plugin are:
 
+* [Embed Optimizer](https://wordpress.org/plugins/embed-optimizer/)
 * [Image Placeholders](https://wordpress.org/plugins/dominant-color-images/)
+* [Image Prioritizer](https://wordpress.org/plugins/image-prioritizer/)
 * [Modern Image Formats](https://wordpress.org/plugins/webp-uploads/)
+* [Optimization Detective](https://wordpress.org/plugins/optimization-detective/) (dependency for Embed Optimizer and Image Prioritizer)
```

Comment:
I thought it would be good to include Optimization Detective in the list here in the readme for visibility.